ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
+
+import AppLayouts.Chat.panels.communities 1.0
+
+StatusModal {
```

Comment:
... is deprecated :_) `StatusDialog` is the new cool thing :)

Reply:
Updated!